src/test/java/org/openmrs/contrib/qaframework/automation/VitalsSteps.java

```diff
@@ -0,0 +1,74 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.reference.page.FindPatientPage;
+import org.openmrs.reference.page.PatientCaptureVitalsPage;
+import org.openmrs.uitestframework.test.TestData;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+
+public class VitalsSteps extends Steps {
+
+	private PatientCaptureVitalsPage patientCaptureVitalsPage;
+	private TestData.PatientInfo testPatient;
+
+	@Before(RunTest.HOOK.SELENIUM_VITALS)
+	public void visitPatientDashboard() {
+	  initiateWithLogin();
+	  testPatient = createTestPatient();
+	  findPatientPage = (FindPatientPage) homePage.goToFindPatientRecord().waitForPage();
+	  findPatientPage.enterPatient(testPatient.identifier);
+	  dashboardPage = findPatientPage.clickOnFirstPatient();
+	  dashboardPage.startVisit().waitForPage();
+	}
+
+	@After(RunTest.HOOK.SELENIUM_VITALS)
+	public void destroy() {
+	  quit();
+	}
+
+	@Given("a user clicks on Capture Vitals link from Patient dashboard")
+	public void launchVitalsPage() {
+	  patientCaptureVitalsPage = (PatientCaptureVitalsPage) dashboardPage.goToPatientCaptureVitalsPage();
+	}
+
+	@Then("the system loads Vitals page")
+	public void systemloadsVitalsPage() {
+	  assertEquals(getElement(patientHeaderId).getText(), getElement(patientHeaderId).getText());
```

Comment:
These need to be two different methods,one method should be the expected and the actual method feel free to [checkout](https://www.browserstack.com/guide/verify-and-assert-in-selenium)